lib/components/Editor/index.js

```diff
@@ -51,6 +52,8 @@ const Tiptap = (
   const isSlashCommandsActive = !hideSlashCommands;
   const showSlashCommandPlaceholder =
     !isPlaceholderActive && isSlashCommandsActive;
+  const isUnsplashImageUploadActive = addons.includes("image-upload-unsplash");
+  isUnsplashImageUploadActive && addons.push("image-upload");
```

Comment:
IMO We shouldn't be directly modifying an array passed down as prop. We should be cloning the values to a new array and push the new value. Otherwise, we are directly modifying a value/data from the library user's code since the Arrays are passed by reference.